v2/brigadier/README.md

```diff
@@ -0,0 +1,98 @@
+# Brigadier: The JS/TS library for Brigade
+
+Brigadier is the library for writing [Brigade](https://brigade.sh) scripts.
+
+It provides core Brigade objects such as Events and Jobs that developers
+can use to construct expressive event-handling logic for their Brigade
+projects.
+
+Brigadier supports writing scripts in either JavaScript or TypeScript.
+
+## Installation
+
+[![NPM](https://nodei.co/npm/@brigadecore/brigadier.png)](https://www.npmjs.com/package/@brigadecore/brigadier)
+
+Normally, the brigadier dependency is declared at the top of a Brigade script
+and will be installed when the Brigade Worker starts up before script
+execution, like so:
+
+```javascript
+const { events, Job } = require("@brigadecore/brigadier");
+```
+
+You may also install brigadier to your environment with Yarn, NPM, etc.:
+
+```console
+$ yarn add @brigadecore/brigadier
+```
+
+## Versioning
+
+The 0.x Brigadier npm releases are compatible with
+[Brigade v1.x and under](https://github.com/brigadecore/brigade/tree/master).
+
+The 2.x brigadier npm releases are compatible with
+[Brigade v2.x](https://github.com/brigadecore/brigade/tree/v2).
+
+## Usage
```

Comment:
These seem to be v2 examples. After acknowledging in the previous section that different versions of Brigadier are compatible with different versions of Brigade, perhaps it's incumbent to show examples here for both?

Reply:
Or if we _don't_ want to show for both, should we at least clarify that these are v2 examples?